constants/application.ts

```
@@ -1,7 +1,8 @@
 const APPLICATION_STATUS_TYPES = ["accepted", "rejected", "pending"];
+const NEW_APPLICATION_STATUS_TYPES = ["not_submitted", "pending", "accepted", "rejected", "requested_changes"]
```

ESLint

[error] Insert `;`

(prettier/prettier)